### PR TITLE
Added iuphar to script

### PR DESCRIPTION
**Pull request by DeniseSl22**
@egonw , could you please check if I added the IUPHAR database and it's IDs in the correct manner? I still commented everything in the script, so it will not break the Jenkins jobs (before I can test it).... but then I know I'm on the right track :).

### CODE REVIEW

File: createDerby.groovy [hunk 1]
```diff
@@ -54,6 +54,7 @@ lmDS = DataSource.register ("Lm", "LIPID MAPS").asDataSource()
 knapsackDS = DataSource.register ("Cks", "KNApSAcK").asDataSource()
 dtxDS = DataSource.register ("Ect", "EPA CompTox").asDataSource()
 // drugbankDS = BioDataSource.DRUGBANK
+//iupharDS = DataSource.register ("Gpl", "IUPHAR").asDataSource()
```
Comment (egonw) [inline createDerby.groovy]: Change IUPHAR to the name used in the `datasources.txt`.

File: createDerby.groovy [hunk 2]
```diff
@@ -184,6 +185,8 @@ if (hmdbFile.exists()) {
          addXRef(database, ref, rootNode.cas_registry_number.toString(), casDS, genesDone, linksDone);
          addXRef(database, ref, rootNode.pubchem_compound_id.toString(), pubchemDS, genesDone, linksDone);
          addXRef(database, ref, rootNode.chemspider_id.toString(), chemspiderDS, genesDone, linksDone);
+	//   addXRef(database, ref, rootNode.iuphar_id, iupharDS,
```
Comment (egonw) [inline createDerby.groovy]: I think this section is where it added stuff from the HMDB database. I don't think we want that...

File: createDerby.groovy [hunk 3]
```diff
@@ -430,6 +435,37 @@ new File("cs2wikidata.csv").eachLine { line,number ->
 }
 unitReport << "  <testcase classname=\"WikidataCreation\" name=\"ChemSpiderFound\"/>\n"
 
+//// IUPHAR registry numbers
```
Comment (egonw) [inline createDerby.groovy:435]: This is the only place where we should have iupharDS:

**Review by egonw — CHANGES_REQUESTED** [review]
I think you only need that first line (where you create the DataSource object) and the last big code block. The rest is reading the HMDB and ChEBI content.

File: createDerby.groovy (hunk 1 above)
Comment (DeniseSl22) [reply createDerby.groovy]: Yes I will!

File: createDerby.groovy (hunk 2 above)
Comment (DeniseSl22) [reply createDerby.groovy]: Okay, makes sense now. I'll remove it.

File: createDerby.groovy (hunk 3 above)
Comment (DeniseSl22) [reply createDerby.groovy:435]: So only reading in from Wikidata, got it.

**Comment by DeniseSl22** [comment]
Addressed Egon's issues raised above with new push.

**Pull request closed without merging**